src/eko/evolution_operator/grid.py

```diff
@@ -94,17 +82,6 @@ def __init__(
         config["polarized"] = configs.polarized
         config["time_like"] = configs.time_like
 
-        if method not in [
-            "iterate-exact",
-            "iterate-expanded",
-            "truncated",
-            "ordered-truncated",
-            "decompose-exact",
-            "decompose-expanded",
-            "perturbative-exact",
-            "perturbative-expanded",
-        ]:
-            raise ValueError(f"Unknown evolution mode {method}")
```

Comment:
why this check has been removed?

Reply:
Full explanation in the commit message: https://github.com/NNPDF/eko/commit/93e662d5c311377df6ec01f8cdd06b1855b91101